client/chat/channel-info-card.tsx

```diff
@@ -92,42 +138,35 @@ export function ConnectedChannelInfoCard({
     setIsJoinInProgress(true)
     dispatch(
       joinChannel(channelName, {
-        onSuccess: channel => navigateToChannel(channel.id, channel.name),
+        onSuccess: () => {},
         onError: err => {
-          setJoinChannelError(err)
           setIsJoinInProgress(false)
+
+          // NOTE(2Pac): We assume the rest of the errors are handled elsewhere since we only care
```

Comment:
`We assume the error has been handled in <ACTUAL PLACE IT IS BEING HANDLED>, we just need to deal with the ban case in this UI`